assets/sass/patterns/organisms/site-header.scss

```diff
@@ -76,6 +76,30 @@
   z-index: 20;
 }
 
+.skip-to-content {
+  display: block;
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  white-space: nowrap;
+}
+
+.skip-to-content__link {
+  position: absolute;
+  top: -100px;
+  left: 0;
+  overflow: hidden;
+  z-index: 50;
+
+  &:focus {
```

Comment:
The corollary of `visuallyhidden` is `visuallyunhidden` (see https://github.com/elifesciences/pattern-library/blob/develop/assets/sass/_mixins.scss#L151). Please use this when reversing the effects of visuallyhidden